lib/firespring_dev_commands/aws/route53.rb

```diff
@@ -0,0 +1,72 @@
+require 'aws-sdk-route53'
+
+module Dev
+  class Aws
+    # Class for performing Route53 functions
+    class Route53
+      attr_reader :client, :zones
+
+      def initialize
+        @client = ::Aws::Route53::Client.new
+        @zones = []
+      end
+
+      def get_hosted_zones(domains)
+        if domains.empty?
+          response = client.list_hosted_zones
```

Comment:
The max returned is 100 which isn't really going to work for us - we'll have to page through all of them. I wrote an `each_page` helper in aws.rb - but the pagination for route53 seems a little different so I'm not sure it will work out of the box. You should be able to set the `max_items` to "2" or something to test the pagination